test/controllers/nonsipp/otherassetsdisposal/TypeOfAssetBuyerControllerSpec.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.nonsipp.otherassetsdisposal
+
+import config.Refined.{Max50, Max5000}
+import controllers.ControllerBaseSpec
+import controllers.nonsipp.otherassetsdisposal.TypeOfAssetBuyerController._
+import eu.timepit.refined.refineMV
+import forms.RadioListFormProvider
+import models.IdentityType.{Individual, Other, UKCompany, UKPartnership}
+import models.NormalMode
+import views.html.RadioListView
+
+class TypeOfAssetBuyerControllerSpec extends ControllerBaseSpec {
+
+  private val assetIndex = refineMV[Max5000.Refined](1)
+  private val disposalIndex = refineMV[Max50.Refined](1)
+
+  private lazy val onPageLoad =
+    routes.TypeOfAssetBuyerController.onPageLoad(srn, assetIndex, disposalIndex, NormalMode)
+  private lazy val onSubmit =
+    routes.TypeOfAssetBuyerController.onSubmit(srn, assetIndex, disposalIndex, NormalMode)
+
+  "WhoWereTheSharesSoldToController" - {
```

Comment:
Not important enough to block the ticket, but this still says 'Shares'

Reply:
Yes Ben, no worries will fix it